Makefile

```diff
@@ -374,7 +375,13 @@ presubmit_dashboard: descriptions
 presubmit_race: descriptions
 	# -race requires cgo
 	env CGO_ENABLED=1 $(GO) test -race; if test $$? -ne 2; then \
-	env CGO_ENABLED=1 $(GO) test -race -short -vet=off -bench=.* -benchtime=.2s ./... ;\
+	env CGO_ENABLED=1 SYZ_SKIP_DEV_APPSERVER_TESTS=1 $(GO) test -race -short -vet=off -bench=.* -benchtime=.2s ./... ;\
+	fi
+
+presubmit_race_dashboard: descriptions
+	# -race requires cgo
+	env CGO_ENABLED=1 $(GO) test -race; if test $$? -ne 2; then \
```

Comment:
Why do we need this line?

Reply:
IIRC this allows to run "make presubmit" on platforms that don't have -race support.
Commit message may have more info.

Reply:
It checks for the -race support.
If -race is not supported, go test -race returns error code 2.

Was added by Greg Steuck in 2018. You can guess what OS missed -race that time.